spinta/datasets/commands/check.py

```diff
@@ -1,14 +1,45 @@
 from spinta import commands
 from spinta.components import Context
 from spinta.datasets.components import Dataset, Resource
+import re
 
 
 @commands.check.register()
 def check(context: Context, dataset: Dataset):
     for resource in dataset.resources.values():
         commands.check(context, resource)
 
+    rc = context.get('rc')
+
+    if rc.get('names'):
```

Comment:
```suggestion
    config: Config = context.get('config')

    if config.check_names:
```

All configuration parameters should be set on top of `config`.

See: https://github.com/atviriduomenys/spinta/blob/master/spinta/types/config.py